Integrations/CounterCraft/CounterCraft.yml

```diff
@@ -0,0 +1,420 @@
+      description: Host Description
+      type: string
+    - contextPath: CounterCraft.Host.StatusCode
+      description: Host Status
+      type: string
+    - contextPath: CounterCraft.Host.TypeCode
+      description: Host Type
+      type: string
+    description: Lists all deception hosts
+  - name: countercraft-list-services
+    arguments:
+    - name: host_id
+      description: Host Id
+    outputs:
+    - contextPath: CounterCraft.Service.ID
+      description: Service ID
+      type: number
+    - contextPath: CounterCraft.Service.Name
+      description: Service Name
+      type: string
+    - contextPath: CounterCraft.Service.Description
+      description: Service.Description
+      type: string
+    - contextPath: CounterCraft.Service.StatusCode
+      description: Service Status
+      type: string
+    - contextPath: CounterCraft.Service.TypeCode
+      description: Service Type
+      type: string
+    description: 'List services currently deployed on deception hosts'
+  - name: countercraft-list-breadcrumbs
+    arguments:
+    - name: campaign_id
+      description: Campaign ID
+    outputs:
+    - contextPath: CounterCraft.Breadcrumb.ID
+      description: Breadcrumb ID
+      type: number
+    - contextPath: CounterCraft.Breadcrumb.Name
+      description: Breadcrumb Name
+      type: string
+    - contextPath: CounterCraft.Breadcrumb.Description
+      description: Breadcrumb Description
+      type: string
+    - contextPath: CounterCraft.Breadcrumb.StatusCode
+      description: Breadcrumb Status
+      type: string
+    - contextPath: CounterCraft.Breadcrumb.TypeCode
+      description: Breadcrumb Type
+      type: string
+    description: List breadcrumbs in a campaign
+  - name: countercraft-get-object
+    arguments:
+    - name: value
+      required: true
+      description: Object value
+    outputs:
+    - contextPath: CounterCraft.Object.ID
+      description: Object ID
+      type: number
+    - contextPath: CounterCraft.Object.Value
+      description: Object value
+      type: string
+    - contextPath: CounterCraft.Object.Hits
+      description: Object hits
+      type: number
+    - contextPath: CounterCraft.Object.Score
+      description: Object score
+      type: number
+    - contextPath: CounterCraft.Object.TypeCode
+      description: Object type
+      type: string
+    - contextPath: CounterCraft.Object.FirstSeen
+      description: Object first seen
+      type: date
+    - contextPath: CounterCraft.Object.LastSeen
+      description: Object last seen
+      type: date
+    - contextPath: CounterCraft.Object.EventsCount
+      description: Object events count
+      type: number
+    - contextPath: CounterCraft.Object.Tags
+      description: Object tags
+      type: string
+    description: Get information about an object (IoC)
+  - name: countercraft-get-events
+    arguments:
+    - name: criteria
+      required: true
+      description: Search criteria
```

Comment:
please add an example to the description

Reply:
I added an example in the documentation in all commands, like the following. Do I need to also add it to the description field?

##### Command Example
```!countercraft-get-events criteria="type_code:ValidAuth" max_results="1"```

Reply:
Done